docs/content/set-up-new-project.md

```diff
@@ -8,6 +8,16 @@ Follow the [Setup and installation guide](../setup-and-installing.md) to make su
 
 To use the SpatialOS GDK for Unity in a new project, you need to set up your project manifest, and then set up world initialization. Your new project must have the [same structure as a SpatialOS project](https://docs.improbable.io/reference/latest/shared/reference/project-structure).
 
+The **spatialos.json** file for your project needs to have the same `sdk_version` and `dependencies` as the GDK's [spatialos.json](../../spatialos.json).
+
+For a basic set up of two workers, one `UnityGameLogic` and one `UnityClient`, we recommend you to reuse these files (within `workers/unity`):
+- [spatialos.UnityGameLogic.worker.json](../../workers/unity/spatialos.UnityGameLogic.worker.json)
+- [spatialos.UnityClient.worker.json](../../workers/unity/spatialos.UnityClient.worker.json)
+
+### Set up base assets and directories
+- Copy [this file](../../workers/unity/Assets/Config/GdkToolsConfiguration.json) into `workers/unity/Assets/Config/GdkToolsConfiguration.json`.
```

Comment:
interesting, though this would be automatically generated? need to check with Jamie for that

Reply:
Adding @jamiebrynes7 ☝️

Reply:
I had an error when I didn't have it, but can check what happens if I follow everything else but this

Reply:
This should be auto generated. If its not that's a bug report

Reply:
Just deleted it and can confirm it was indeed re-generated